openapi/Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM maven:3.5-jdk-7
+ARG SWAGGER_CODEGEN_COMMIT
+ARG GENERATION_XML_FILE
+
+# Install preprocessing script requirements
+RUN apt-get update && apt-get -y install python-pip && pip install urllib3
+
+# Check out specific commit of swagger-codegen
+RUN mkdir /source && \
+    cd /source && \
+    git clone -n https://github.com/swagger-api/swagger-codegen.git && \
```

Comment:
I think this should take the user/org as an optional parameter as well. It would make it easier for someone developing against swagger codegen to see how their changes build the client by just pushing their changes to git first.